src/build.py

```diff
@@ -917,7 +918,7 @@ def LLVM(build_dir):
         targets = ['clang', 'lld', 'llvm-ar', 'llvm-addr2line', 'llvm-cxxfilt',
                    'llvm-dwarfdump', 'llvm-dwp', 'llvm-nm', 'llvm-objcopy',
                    'llvm-objdump', 'llvm-ranlib', 'llvm-readobj', 'llvm-size',
-                   'llvm-strings', 'llvm-symbolizer']
+                   'llvm-strings', 'llvm-symbolizer', 'clang-resource-headers']
```

Comment:
Why do we have this list for lto but not for the normal build?

Reply:
Because the LTO build uses the `distribution` ninja build target instead of the default `all` target (see the next line, 922). We could (and maybe should) do this for the non-LTO build as well, but I figured I'd wait until the LTO build was stable before trying to merge them.